Notes/implementation-notes.md

```diff
@@ -189,9 +190,9 @@ we'll see that the @-pattern name is defined, so turn it into a PLet.
 Names which are bound in types are also bound as @-patterns, meaning that
 functions have access to them. For example, we can say:
 
-vlength : Vect n a -> Nat
-vlength [] = n
-vlength (x :: xs) = n
+    vlength : Vect n a -> Nat
+    vlength [] = n
+    vlength (x :: xs) = n
```

Comment:
Perhaps you could use fences, like:

````md
```idris
vlength : Vect n a -> Nat
vlength [] = n
vlength (x :: xs) = n
```
````

Reply:
Good idea, thanks!